web/vasilemaftita/html/css/syntax one/index.html

```diff
@@ -5,14 +5,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Syntax one</title>
+    <link rel="stylesheet" href="syntaxone.css" />
```

Comment:
Here you're importing "syntaxone.css" and this file doesn't exist. Or at least not in this PR. Can you double check if you have it please and if not, this line can be removed. Thx